cmd/create/cluster/cmd.go

```diff
@@ -991,6 +991,66 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
+	cloudAccounts, err := r.OCMClient.GetBillingAccounts()
+	if err != nil {
+		r.Reporter.Errorf("%s", err)
+		os.Exit(1)
+	}
+
+	billingAccounts := ocm.GenerateBillingAccountsList(cloudAccounts)
+
+	if billingAccount == "" && !interactive.Enabled() {
+		// if a billing account is not provided we will try to use the infrastructure account as default
+		if helper.ContainsPrefix(billingAccounts, awsCreator.AccountID) {
+			billingAccount = awsCreator.AccountID
+			r.Reporter.Infof("Using '%s' as billing account", billingAccount)
+			r.Reporter.Infof("To use a different billing account, add --billing-account xxxxxxxxxx to previous command")
+		} else {
+			r.Reporter.Errorf("A billing account is required for Hosted Control Plane clusters.")
+		}
+	}
+
+	if interactive.Enabled() {
+		if len(billingAccounts) > 0 {
+			billingAccount, err = interactive.GetOption(interactive.Input{
+				Question: "Billing Account",
+				Help:     cmd.Flags().Lookup("billing-account").Usage,
+				Default:  billingAccount,
+				Required: true,
+				Options:  billingAccounts,
+			})
+
+			if err != nil {
+				r.Reporter.Errorf("Expected a valid billing account: '%s'", err)
+				os.Exit(1)
+			}
+
+			billingAccount = ocm.RegexpReturnOnlyInteger(billingAccount)
```

Comment:
The `ParseOption` function can be reused here. See `pkg/aws/helpers.go`

Reply:
Thanks! Replaced with the ParseOption